examples/standalone/optimiser.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+from scipy.optimize import minimize
+
+from pybop import Optimisation
+
+
+class StandaloneOptimiser(Optimisation):
+    """
+    Defines an example standalone optimiser without a Cost.
+    """
+
+    def __init__(self, cost=None, **optimiser_kwargs):
+        # Define cost function
+        def cost(x):
+            x1, x2 = x
+            return (x1 - 2) ** 2 + (x2 - 4) ** 4
```

Comment:
I'm not sure what you mean? This StandaloneOptimiser can be run without an instance of the BaseCost. Without these lines defining the cost function, it cannot be run independently.

Reply:
Right, I think the standalone optimiser should be without the addition of a cost function within the class. Let's discuss this as a follow on though, as I realise it would require either skipping the file in the --examples tests or moving these standalones outside of the `examples' directory.

Reply:
I think it is consistent with the other Standalone examples for it to be defined this way, but yes, let's discuss further after this PR.